server/src/main/java/org/opensearch/cluster/routing/OperationRouting.java

```diff
@@ -79,6 +82,9 @@ public class OperationRouting {
     private volatile boolean useAdaptiveReplicaSelection;
     private volatile boolean ignoreAwarenessAttr;
 
+    public WRRShardsCache wrrShardsCache;
+    public ClusterService clusterService;
```

Comment:
can we make this private and final please

Reply:
Making this private. Can't make it final since we are injecting the values.